.cursor/prompts/코드스타일_가이드라인.md

```
@@ -0,0 +1,1213 @@
+### 🚀 프로젝트 API 특이사항 (매우 중요!)
+
+#### 기본 원칙
+
+- **백엔드 동기화**: `src/types/api/` 폴더의 타입 정의는 백엔드 객체와 완벽 동기화
+- **FormData 통신**: 거의 모든 API가 POST + FormData 형식으로 통신
+- **Command/Dto 패턴**: 요청시 Command 접미사, 응답시 Dto 접미사 사용
+- **인증 토큰**: AccessToken(sessionStorage) + RefreshToken(쿠키) 방식
+
+#### API 통신 규칙
+
+**✅ 사용해야 할 폴더**
```

markdownlint-cli2 (0.17.2)

Emphasis used instead of a heading
null

(MD036, no-emphasis-as-heading)